RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple13.java

```diff
@@ -27,6 +27,7 @@
         M extends FileObject
         > extends AbstractFileObjectTuple {
 
+    // TODO: @dankwart-de: Why has Tuple13 different names for the fields then e.g. Tuple2??
```

Comment:
For that, you need to know, how I named the tuple fields in the first tuple implementations. I took the names from computer graphics vectors, namely x, y, z and w / r (or else). As you can see above, this mechanism gets confusing and misleading for higher order vectors and stops working at some point. That's why I started the tuple13 with _a